src/routes/member/[studentId]/edit-bio/+page.server.ts

```diff
@@ -0,0 +1,60 @@
+import { ctxAccessGuard } from "$lib/utils/access";
+import apiNames from "$lib/utils/apiNames";
+import prisma from "$lib/utils/prisma";
+import { Prisma } from "@prisma/client";
+import { error, fail } from "@sveltejs/kit";
+import type { PageServerLoad } from "./$types";
+
+export const load: PageServerLoad = async ({ params, parent }) => {
+  const member = await prisma.member.findUnique({
+    where: {
+      studentId: params.studentId,
+    },
+  });
+  if (!member) {
+    throw error(404, "Member not found");
+  }
+  const { session } = await parent();
+  await ctxAccessGuard(apiNames.MEMBER.UPDATE, session?.user, { studentId: params.studentId });
+  return {
+    member,
+  };
+};
+
+export const actions = {
+  update: async ({ params, locals, request }) => {
+    const session = await locals.getSession();
+    await ctxAccessGuard(apiNames.MEMBER.UPDATE, session?.user, { studentId: params.studentId });
+    const studentId = params.studentId;
+    const formData = await request.formData();
```

Comment:
formData validation seems a bit repetitive. Do you think we should use zod for form validation?

Reply:
Or [Superform](https://superforms.rocks/)

Reply:
I think this can be changed in a future pr